contracts/Avocado.sol

```diff
@@ -0,0 +1,128 @@
+pragma solidity ^0.4.17;
+
+contract Avocado {
+    enum PersonType { Teacher, Student }
+
+    // Person Structure
+    struct Person {
+        // Teacher or Student
+        PersonType personType;        
+        string name;        
+        string description;                        
+        uint ethPerHour;
+
+        // Array of the Hash of the active meetings
+        bytes32[] activeMeetings;
+
+        // Array of the Hash of completed meetings
+        bytes32[] completedMeetings;
+                
+        bool exists;
+    }
+
+    // Meeting structure
+    struct Meetings {
+        address student;
+        address teacher;
+        string description;
+        uint32 timeInEpoch; // When the session's gonna happen
```

Comment:
`timestamp` or `unixTimestamp` might be easier to understand (Josh didn't know what an Epoch was).

Reply:
Fair point